packages/card/resource-card/index.test.tsx

```diff
@@ -31,4 +31,18 @@ describe('<ResourceCard />', () => {
 
     expect(metaElement).toContainElement(screen.getByText('|'))
   })
+
+  it('should not render the date if no date is provided', () => {
+    defaultProps.date = ''
```

Comment:
Instead of modifying default props, make a new object and spread defaultProps

```
const defaultPropsWithoutDate = { ...defaultProps, date: '' }
```

Reply:
Good call, addressed here:
https://github.com/hashicorp/react-components/pull/1001/commits/dd8b04a63f301a9cdb2a78a4f0c001e0ae784eeb